Traceback (most recent call last):
  File "/home/amax/anaconda3/envs/python3.8/lib/python3.8/site-packages/slither/core/cfg/node.py", line 871, in _find_read_write_call
    self._high_level_calls.append((ir.destination.type.type, ir.function))
AttributeError: 'NoneType' object has no attribute 'type'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/amax/anaconda3/envs/python3.8/lib/python3.8/site-packages/slither/__main__.py", line 837, in main_impl
    ) = process_all(filename, args, detector_classes, printer_classes)
  File "/home/amax/anaconda3/envs/python3.8/lib/python3.8/site-packages/slither/__main__.py", line 101, in process_all
    ) = process_single(compilation, args, detector_classes, printer_classes)
  File "/home/amax/anaconda3/envs/python3.8/lib/python3.8/site-packages/slither/__main__.py", line 79, in process_single
    slither = Slither(target, ast_format=ast, **vars(args))
  File "/home/amax/anaconda3/envs/python3.8/lib/python3.8/site-packages/slither/slither.py", line 135, in __init__
    self._init_parsing_and_analyses(kwargs.get("skip_analyze", False))
  File "/home/amax/anaconda3/envs/python3.8/lib/python3.8/site-packages/slither/slither.py", line 155, in _init_parsing_and_analyses
    raise e
  File "/home/amax/anaconda3/envs/python3.8/lib/python3.8/site-packages/slither/slither.py", line 151, in _init_parsing_and_analyses
    parser.analyze_contracts()
  File "/home/amax/anaconda3/envs/python3.8/lib/python3.8/site-packages/slither/solc_parsing/slither_compilation_unit_solc.py", line 528, in analyze_contracts
    self._convert_to_slithir()
  File "/home/amax/anaconda3/envs/python3.8/lib/python3.8/site-packages/slither/solc_parsing/slither_compilation_unit_solc.py", line 739, in _convert_to_slithir
    func.generate_slithir_and_analyze()
  File "/home/amax/anaconda3/envs/python3.8/lib/python3.8/site-packages/slither/core/declarations/function.py", line 1761, in generate_slithir_and_analyze
    node.slithir_generation()
  File "/home/amax/anaconda3/envs/python3.8/lib/python3.8/site-packages/slither/core/cfg/node.py", line 681, in slithir_generation
    self._find_read_write_call()
  File "/home/amax/anaconda3/envs/python3.8/lib/python3.8/site-packages/slither/core/cfg/node.py", line 874, in _find_read_write_call
    raise SlitherException(
slither.exceptions.SlitherException: Function not found on IR: TMP_1011(None) = HIGH_LEVEL_CALL, dest:TMP_1010(None), function:sub, arguments:['1']  .
Node: EXPRESSION scIndex = (stakedContractStakers[_stakedContractAddress].push(Staker(_stakerAddress,stakerStakedContracts[_stakerAddress].length))).sub(1) (../../sc/dataset/mainnet/0xcafea677DB7Ab72D3Ae599E2747f3e944bE1d718.sol#3379-3380)
Function: addStake
Please try compiling with a recent Solidity version. 'NoneType' object has no attribute 'type'
ERROR:root:Error:
ERROR:root:Function not found on IR: TMP_1011(None) = HIGH_LEVEL_CALL, dest:TMP_1010(None), function:sub, arguments:['1']  .
Node: EXPRESSION scIndex = (stakedContractStakers[_stakedContractAddress].push(Staker(_stakerAddress,stakerStakedContracts[_stakerAddress].length))).sub(1) (../../sc/dataset/mainnet/0xcafea677DB7Ab72D3Ae599E2747f3e944bE1d718.sol#3379-3380)
Function: addStake
Please try compiling with a recent Solidity version. 'NoneType' object has no attribute 'type'
ERROR:root:Please report an issue to https://github.com/crytic/slither/issues
